goldstone/core/tests.py

```diff
@@ -93,6 +98,254 @@ def test_delete_indices(self):
         self.assertEqual(tasks.delete_indices('abc', 10), 'mocked')
 
 
+class DailyIndexDocTypeTests(SimpleTestCase):
+
+    class CADFEvent(DailyIndexDocType):
```

Comment:
Could probably remove this custom class and use the one that's in core/models now.